Gemfile.lock

```diff
@@ -11,9 +11,9 @@ GEM
       faye-websocket (~> 0.11.0)
       selenium-webdriver (~> 4.2, < 4.6)
     atomos (0.1.3)
-    bugsnag (6.25.2)
+    bugsnag (6.26.0)
```

Comment:
All of the changes in the Gemfile.lock will be revoked, as no changes are necessary in this file